app/components/common/list/index.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import {
+  View,
+  ScrollView,
+  Image,
+  Text
+} from 'react-native';
+import styles from './styles';
+
+const imageExample = require('assets/appicon/app-icon@120x120.png');
+const nextImage = require('assets/next@2x.png');
```

Comment:
For images let's use without @2x so the system can choose depending on the device =)